appveyor.yml

```diff
@@ -0,0 +1,21 @@
+environment:
+  matrix:
+    - RUBY_VERSION: 22
```

Comment:
Worth nothing this means Ruby 2.2.6p396

Reply:
At time of writing, yeah.  I suspect Appveyor will update the versions, but keep 22 meaning Ruby 2.2.xxx and 21 meaning Ruby 2.1.xxx